jax/lax/lax.py

```diff
@@ -1603,6 +1611,16 @@ def binop(result_dtype, accepted_dtypes, name, translation_rule=None):
   return prim
 standard_binop = partial(binop, _input_dtype)
 
+def terop(result_dtype, accepted_dtypes, name, translation_rule=None):
```

Comment:
If you merge the ternary and binary cases in `masking.py`, these two rules are then identical and can be merged, I think?